test/services/objection.service.spec.unit.ts

```diff
@@ -132,6 +133,11 @@ describe("objections API service unit tests", () => {
         ATTACHMENT_ID);
     expect(attachment).toEqual(mockAttachment);
   });
+
+  it("should call sdk when deleting attachment", async () => {
+    await objectionsService.deleteAttachment(session, ATTACHMENT_ID);
+    expect(mockDeleteAttachment).toBeCalledTimes(1);
```

Comment:
I would also check the correct attachmentId is passed as well, try toBeCalledWith or hasBeenCalledWith